docs/source/hardware/partitions.rst

```diff
@@ -16,13 +16,25 @@ distribution of resources for all jobs being submitted to the cluster.
 
 On Discovery, there are several different partitions:
 
-* General access (``debug``, ``express``, ``short``, ``gpu``)
+* General access (``debug``, ``express``, ``short``, ``gpu``, ``lowpriority``)
 * Application only (``long``, ``large``, ``multigpu``)
-* PI owned (accessed only by members of the PIs’ group)
+* PI owned (accessed only by members of the PIs’ group unless the PI partition is a member of the ``lowpriority`` partition group)
+
+The general access and application only partitions span a variety of
+hardware on Discovery. The ``gpu`` and ``multigpu`` partitions are
+meant for accesssing the GPU hardware on Discovery while the rest make
+CPUs available to the community.  For example, if you use the
+``debug`` partition you're using the same hardware as ``short``, just
+with different time, job, and core limits. Refer to the tables below
+for detailed information on the current partitions. 
+
+.. note::
+PI-owned partitions only include the hardware that those PIs own and
+are 'only' accessible to the members of the PI's group unless the
+PI-owned partition is also a member of the ``lowpriority`` partition
+group, in which case it could be available to general users as
+well. See `Low Priority Partition < >`_ .
```

Comment:
Should <_> be included in the link?

Reply:
@joseph-cooke , <> is a placeholder for putting the actual URL of the low priority partition page, once that's finalized. Maybe I can just type a dummy URL within the angled-brackets to avoid confusion.

Reply:
@joseph-cooke, are we good here?